src/data/python-quiz.ts

```diff
@@ -1,4 +1,37 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "In Python, what keyword is used to perform Bitwise NOT operation?",
+    Answer: "~",
+    Distractor1: "NOT",
+    Distractor2: "!",
+    Distractor3: "not",
+    Explanation: 
+      "The and keyword is used in Python to perform Bitwise NOT operations.",
+    Link: "https://www.freecodecamp.org/news/basic-operators-in-python-with-examples/",
+  },
+  {
+    Question: 
+      "In Python, what keyword is used to perform Bitwise OR operation?",
+    Answer: "|",
+    Distractor1: "OR",
+    Distractor2: "or",
+    Distractor3: "||",
+    Explanation: 
+      "The or keyword is used in Python to perform Bitwise OR operations.",
+    Link: "https://www.freecodecamp.org/news/basic-operators-in-python-with-examples/",
+  },
+  {
+    Question: 
+      "In Python, what keyword is used to perform Bitwise AND operation?",
+    Answer: "&",
+    Distractor1: "AND",
+    Distractor2: "&&",
+    Distractor3: "&",
+    Explanation: 
+      "The and keyword is used in Python to perform Bitwise AND operations.",
```

Comment:
Based on the answer you supplied I think the question should be which operator is used for that bitwise operation.
Let's go with this change

```suggestion
      "In Python, which operator is used to perform Bitwise AND operations?",
    Answer: "&",
    Distractor1: "{{",
    Distractor2: "&&",
    Distractor3: "<",
    Explanation: 
      "The & operator is used in Python to perform Bitwise AND operations.",
```